rllib/BUILD.bazel

```diff
@@ -82,28 +82,54 @@ doctest(
 )
 
 # --------------------------------------------------------------------
-# Algorithms learning regression tests.
+# Algorithms learning regression tests (rllib/examples/algorithm/[algo-name]).
 #
 # Tag: learning_tests
 #
-# This will test python/yaml config files
-# inside rllib/examples/algorithms/[algo-name] for actual learning success.
+# These tests check that the algorithm achieves above random performance within a relatively short period of time,
+#   not that the algorithm reaches the optimal policy.
+#
+# For single to multi-learner tests, the expected output should change,
+#   either reducing the maximum iterations or samples, or increasing the max return
+#   to ensure that the multi-learner is achieving something that the single shouldn’t be able to normally achieve.
+#
+# Compute Config
+#   - local (CPU) = 7 CPUs, 0 GPU, 5 Env Runners, 0 Learners on CPU
+#   - single (CPU) = 7 CPUs, 0 GPU, 5 Env Runner, 1 Learners on CPU
+#   - single (GPU) = 7 CPUs, 1 GPU, 5 Env Runner, 1 Learners on GPU
+#   - multi (GPU) = 7 CPUs, 2 GPUs, 5 Env Runners, 2 Learners on GPU
```

Comment:
For 2 learners I'd do:
`multi GPU = 14 CPUs, 2 GPUs, 10 Env Runners`

I want to somehow indicate scale, even on premerge.